.github/workflows/csharp.yaml

```diff
@@ -63,11 +63,10 @@ jobs:
             with:
               dotnet-version: ${{ matrix.dotnet-version }}
 
-          - name: Install dependencies

```

Comment:
@kotbegemot @OrangeSmokingJacket so we don't need those dependences?

Reply:
There was only dotnet restore and dotnet test run it implicitly (https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-restore)